.github/workflows/workflow.yml

```diff
@@ -14,15 +14,15 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: Setup clang-format
+      env:
+        CLANG_VERSION: '12.0.0'
       run: |
-        wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.1/clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz
-        tar -xvf clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz
-        sudo mv ./clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04/bin/clang-format /usr/local/bin
-        rm -rf clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04/
+        wget https://github.com/llvm/llvm-project/releases/download/llvmorg-${CLANG_VERSION}/clang+llvm-${CLANG_VERSION}-x86_64-linux-gnu-ubuntu-20.04.tar.xz
```

Comment:
can we extract this into a script like `bazel/install-clang-ci.sh`? 

This way, it's easier to reproduce the same environment locally (if someone need to debug the CI); and
we don't have to duplicate the `wget` commands in line 20 and line 49.